include/cudaq/Frontend/nvqpp/ASTBridge.h

```diff
@@ -580,7 +582,7 @@ class QuakeBridgeVisitor
 /// drives the process of walking the Clang AST and translate pertinent nodes to
 /// an MLIR Op tree containing Quake, CC, and other MLIR dialect operations.
 /// This Action will generate this MLIR Module and rewrite the input source code
-/// (using the Clang Rewriter system) to define quantum kernels as extern.
+/// (using the Clang `Rewriter` system) to define quantum kernels as `extern`.
```

Comment:
This must be an old comment from way back when. We don't use the Rewriter in the bridge right?

Reply:
I think we replaced all that with the fixup linkage postprocessing.